roles/kubernetes-apps/helm/templates/helm-container.j2

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+{{ docker_bin_dir }}/docker run --rm \
```

Comment:
helm init doesn't include this option. I'll make an upstream bug in helm to add this